src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,40 @@
+import { GameCard } from "./components/Gamecard";
+import "./home.css";
+import { CATEGORIES } from "../../constants/categories";
+
+function Home() {
+	return (
+		<div>
+			<div className="landingpage">
+				<picture>
+					<source
+						media="(min-width: 1000px)"
+						srcset="https://res.cloudinary.com/ecommerce-muskaan/image/upload/v1647541607/e-commerce/landingpagedesktop_hta9tn.jpg"
+					/>
+					<img
+						src="https://res.cloudinary.com/ecommerce-muskaan/image/upload/v1647541607/e-commerce/landingpage_gobzt1.jpg"
+						className="img-responsive"
+						alt="landingpage"
+					/>
+				</picture>
+			</div>
+			<h1>Categories</h1>
+			<div className="categories mx-auto">
+				{CATEGORIES.map(category => (
+					<GameCard category={category} />
```

Comment:
whenever rendering elements using through a map, Always provide it with a key attribute to uniquely identify them.